### PR TITLE
avatar 404 silencing

### PR DESCRIPTION
**Pull request by danethurber**
fixes: #204

### CODE REVIEW

File: packages/avatar/.babelrc [hunk 1]
```diff
@@ -2,10 +2,13 @@
   "presets": [
```
Comment (danethurber) [inline packages/avatar/.babelrc]: lots of formatting/linting in this pr

File: packages/avatar/src/js/index.js [hunk 2]
```diff
@@ -22,8 +24,11 @@ export const getColorByName = name => {
 
 export const transformSrc = src => {
   if (!src.match(/gravatar/)) return src
+
   const sep = src.match(/\?/) ? '&' : '?'
-  return `${src}${sep}d=${defaultGravatarImage}`
+  const fallback = encodeURIComponent(fallbackPixel)
```
Comment (danethurber) [inline packages/avatar/src/js/index.js]: gravatar wants the default image path url encoded

File: packages/avatar/src/react/index.js [hunk 3]
```diff
@@ -23,12 +23,16 @@ class Avatar extends React.Component {
     this.handleImageLoadSuccess = this.handleImageLoadSuccess.bind(this)
     this.handleImageLoadError = this.handleImageLoadError.bind(this)
   }
-  handleImageLoadSuccess() {
-    this.setState({ imageState: 'success' })
+
+  handleImageLoadSuccess(event) {
+    const isFallbackPixel = event.target.naturalWidth === 1
```
Comment (danethurber) [inline packages/avatar/src/react/index.js]: this is where the magic happens. our gravatar fallback is 1px wide

File: packages/avatar/package.json [hunk 4]
```diff
@@ -46,7 +47,6 @@
     "babel-preset-stage-2": "^6.24.1",
     "jest": "^23.6.0",
     "npm-run-all": "^4.1.2",
-    "raf": "^3.4.0",
-    "react": "^16.2.0"
```
Comment (jaketrent) [inline packages/avatar/package.json]: awesome, how did you determine you could remove this?

File: packages/avatar/src/js/index.js [hunk 5]
```diff
@@ -1,12 +1,14 @@
-import { colorByLetter, defaultGravatarImage } from '../vars'
+/* eslint-disable camelcase */
```
Comment (jaketrent) [inline packages/avatar/src/js/index.js]: eh?  what's this one for?  I love camelCase.

**Pull request merged**